Zappy_IA_Report/DTC/Code/models.py

```
@@ -0,0 +1,16 @@
+class iaParser:
+    """
+    Represents a iaParser body.
+    """
+    def __init__(self, port, name, machine):
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)